.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "shared-modules"]
```

Comment:
You can now remove this. (assuming it builds)

Reply:
Haha, will I ever not forget the comma?